From: Jiri Slaby <jslaby@suse.cz>
Date: Wed, 19 Aug 2026 08:23:42 +0200
Subject: vm_sockets: include sys/socket.h
References: build-fix
Patch-mainline: Submitted https://github.com/kernelslacker/trinity/pull/59

This was fixed upstream in 22bbc1dcd0d6 ("vsock/uapi: fix
linux/vm_sockets.h userspace compilation errors"). This is not present
on older kernels like on SL-16.0.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/kernel/vm_sockets.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/kernel/vm_sockets.h b/include/kernel/vm_sockets.h
index 9f300429ac5d..06e88491283b 100644
--- a/include/kernel/vm_sockets.h
+++ b/include/kernel/vm_sockets.h
@@ -1,4 +1,5 @@
 #pragma once
+#include <sys/socket.h>
 #include <linux/vm_sockets.h>
 
 #ifndef SO_VM_SOCKETS_BUFFER_SIZE
-- 
2.55.0

